plat/socionext/synquacer/drivers/scmi/sq_scmi.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2019, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef SQ_SCMI_H
+#define SQ_SCMI_H
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include <sq_common.h>
+
+int scmi_get_draminfo(void *p, struct draminfo *info);
```

Comment:
This can go at the end of scmi.h. Create a section for vendor extensions using code comments and add this there.